h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -628,8 +630,14 @@ public void cv_mainModelScores(int N, ModelMetrics.MetricBuilder mbs[], ModelBui
     }
     mainModel._output._cross_validation_metrics._description = N + "-fold cross-validation on training data (Metrics computed for combined holdout predictions)";
     Log.info(mainModel._output._cross_validation_metrics.toString());
+    mainModel._output._cross_validation_metrics_summary = makeCrossValidationSummaryTable(cvModKeys);
 
-    mainModel._output._cross_validation_metrics_summary = makeCrossValidationSummaryTable(mainModel._output._cross_validation_models);
```

Comment:
it would be good to use the same method, it is useful to see the logging messages (you have in the method deleteCrossValidationModels) especially in the cases when the models were implicitly removed

you can add a private and paremetrized method version of deleteCrossValidationModels to share the code

Reply:
that was my original version indeed, can't remember why I changed this: will restore this.

Reply:
closed